<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chatting</class>
 <widget class="QWidget" name="Chatting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_6" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>786</width>
     <height>414</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget_7" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_8" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frame_8">
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QListWidget" name="showListWidget"/>
             </item>
             <item>
              <widget class="QFrame" name="frame_9">
               <property name="frameShape">
                <enum>QFrame::Box</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_13">
                <item>
                 <widget class="QFontComboBox" name="fontCbx"/>
                </item>
                <item>
                 <widget class="QComboBox" name="sizeCbx">
                  <property name="currentIndex">
                   <number>4</number>
                  </property>
                  <item>
                   <property name="text">
                    <string>8</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>9</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>10</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>11</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>12</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>13</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>14</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>15</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>16</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>17</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>18</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>19</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>20</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>21</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>22</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="boldTBtn_3">
                  <property name="minimumSize">
                   <size>
                    <width>33</width>
                    <height>33</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>加粗</string>
                  </property>
                  <property name="text">
                   <string>...</string>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>:/images/bold.png</normaloff>:/images/bold.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>26</width>
                    <height>26</height>
                   </size>
                  </property>
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="italicTBtn_3">
                  <property name="minimumSize">
                   <size>
                    <width>33</width>
                    <height>33</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>倾斜</string>
                  </property>
                  <property name="text">
                   <string>...</string>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>:/images/italic.png</normaloff>:/images/italic.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>26</width>
                    <height>26</height>
                   </size>
                  </property>
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="underlineTBtn_3">
                  <property name="minimumSize">
                   <size>
                    <width>33</width>
                    <height>33</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>下划线</string>
                  </property>
                  <property name="text">
                   <string>...</string>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>:/images/under.png</normaloff>:/images/under.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>26</width>
                    <height>26</height>
                   </size>
                  </property>
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="colorTBtn_3">
                  <property name="minimumSize">
                   <size>
                    <width>33</width>
                    <height>33</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>更改字体颜色</string>
                  </property>
                  <property name="text">
                   <string>...</string>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>:/images/color.png</normaloff>:/images/color.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>26</width>
                    <height>26</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="saveTBtn_3">
                  <property name="minimumSize">
                   <size>
                    <width>33</width>
                    <height>33</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>保存聊天记录</string>
                  </property>
                  <property name="text">
                   <string>...</string>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>26</width>
                    <height>26</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="clearTBtn_3">
                  <property name="minimumSize">
                   <size>
                    <width>33</width>
                    <height>33</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>清空聊天记录</string>
                  </property>
                  <property name="text">
                   <string>...</string>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>:/images/clear.png</normaloff>:/images/clear.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>26</width>
                    <height>26</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QTextEdit" name="msgTextEdit">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>100</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="usrTbWidget">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <column>
          <property name="text">
           <string>用户名</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_10">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnSend">
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="usrNumLbl_3">
         <property name="text">
          <string>在线人数：0人</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_9">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnClose">
         <property name="text">
          <string>退出</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>518</y>
     <width>786</width>
     <height>0</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_15">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_9" native="true">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>786</width>
     <height>0</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
